.github/workflows/maven-automatic.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         settings-path: ${{ github.workspace }} # location for the settings.xml file
 
     - name: Build with Maven
-      run: mvn -Drevision=1145.vb_cf6cf6ed960 -B package --file pom.xml
+      run: mvn -Dchangelist=1145.vb_cf6cf6ed960 -Drevision=1145.vb_cf6cf6ed960 -B package --file pom.xml
```

Comment:
This brings up the interesting question as to what version do we want the automatically built package to be. 
I would say leaving it as `999999-SNAPSHOT` is a fine choice (as upstream has it)
AKA for the `maven-automatic` job not specifying a `changelist`

Reply:
sounds like a good idea, I'll remove the revision and changelist flags from the automatic build and have the changelist flag on the manual release action.